### PR TITLE
CB-370 added blueprint related integration tests

### PR DESCRIPTION
**Pull request by doktoric**

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/Blueprint.java [hunk 1]
```diff
@@ -76,10 +80,26 @@ public static BlueprintEntity getByName(TestContext testContext, BlueprintEntity
         return entity;
     }
 
-    public static Action<BlueprintEntity> postV2() {
+    public static Action<BlueprintEntity> postV4() {
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/Blueprint.java]: please move this class to the newway/entity/blueprint package

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/BlueprintEntity.java [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
 
 import com.sequenceiq.cloudbreak.api.endpoint.v4.blueprints.requests.BlueprintV4Request;
 import com.sequenceiq.cloudbreak.api.endpoint.v4.blueprints.responses.BlueprintV4Response;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.blueprints.responses.BlueprintV4ViewResponse;
 import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
 
 public class BlueprintEntity extends AbstractCloudbreakEntity<BlueprintV4Request, BlueprintV4Response, BlueprintEntity> {
     public static final String BLUEPRINT = "BLUEPRINT";
 
+    private Collection<BlueprintV4ViewResponse> viewResponses;
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/BlueprintEntity.java]: please move this class to the newway/entity/blueprint package

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/BlueprintDeleteAction.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import static java.lang.String.format;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sequenceiq.it.cloudbreak.newway.BlueprintEntity;
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
+
+public class BlueprintDeleteAction implements Action<BlueprintEntity> {
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/BlueprintDeleteAction.java]: please move this class to the newway/action/blueprint package

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/BlueprintGetAction.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import static java.lang.String.format;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sequenceiq.it.cloudbreak.newway.BlueprintEntity;
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
+
+public class BlueprintGetAction implements Action<BlueprintEntity> {
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/BlueprintGetAction.java]: please move this class to the newway/action/blueprint package

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/BlueprintGetListAction.java [hunk 5]
```diff
@@ -19,10 +19,11 @@
     public BlueprintEntity action(TestContext testContext, BlueprintEntity entity, CloudbreakClient client) throws Exception {
         log(LOGGER, format(" Name: %s", entity.getRequest().getName()));
         logJSON(LOGGER, format(" Blueprint list by workspace request:%n"), entity.getRequest());
-        var blueprints = client.getCloudbreakClient()
-                .blueprintV4Endpoint()
-                .list(client.getWorkspaceId());
-        logJSON(LOGGER, format(" Blueprint list has executed successfully:%n"), blueprints);
+        entity.setViewResponses(
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/BlueprintGetListAction.java]: please move this class to the newway/action/blueprint package

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/BlueprintRequestAction.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+import static java.lang.String.format;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sequenceiq.it.cloudbreak.newway.BlueprintEntity;
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
+
+public class BlueprintRequestAction implements Action<BlueprintEntity> {
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/action/BlueprintRequestAction.java]: please move this class to the newway/action/blueprint package

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/BlueprintTest.java [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+                .withName(blueprintName)
+                .withAmbariBlueprint(VALID_BP)
+                .when(Blueprint.postV4(), key(blueprintName))
+                .when(Blueprint.deleteV4(), key(blueprintName))
+                .then((tc, entity, cc) -> {
+                    assertEquals(blueprintName, entity.getName());
+                    return entity;
+                })
+                .when(Blueprint.listV4())
+                .then(new AssertionV2<BlueprintEntity>() {
```
Comment (gregito) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/BlueprintTest.java]: could you please move the assertion to a new method to make the test itself more readable?

**Pull request merged**